### ISSUE TITLE
ParsingError: pythran-linux2.cfg contains 'ip-all'

### ISSUE THREAD

**Issue by xantares** [serge-sans-paille/pythran#571]
I cannot load pythran after install:
```
In [1]: import pythran
...
ParsingError: File contains parsing errors: /usr/lib/python2.7/site-packages/pythran/pythran-linux2.cfg
        [line 11]: 'ip-all\n'
```

Here's the faulty pythran-linux2.cfg:
```
[compiler]
defines=
undefs=
include_dirs=
libs=
library_dirs=
cflags=-std=c++11 -fno-math-errno -w -fwhole-program -fvisibility=hidden
ldflags=-fvisibility=hidden -Wl,-strip-all
CC=
CXX=
ip-all
CC=
CXX=
```
If I remove the "ip-all" line the module loads fine.

I suspect the end of "strip-all" linker command got here.
Using git version r1767.ede4731, python2.7.11, archlinux.

**Comment by serge-sans-paille** [comment]
Hi @xantares,
how did you install pythran? from pip? from git? Can you provide a minimal list of command, starting with a virtualenv creation, that ends up with this issue?

**Comment by xantares** [comment]
I installed from yaourt package manager (python2-pythran-git), that is basically the git version.
Then just 'import pythran'.

**Comment by serge-sans-paille** [comment]
I've been looking into various location to try to understand the issue, but I cannot reproduce it at all, neither give a clue of its origin :-/

If you run:

```
$ virtualenv tmp
$ source tmp/bin/activate
$ pip install numpy
$ pip install pythran
$ python -c 'import pythran'
```
do you get any issue?

**Comment by xantares** [comment]
Installing using virtualenv was fine (0.7.4 something ?), maybe it's the current git version ?

It's definitely during ```python2 setup.py build``` that this ```build/lib/pythran/pythran-linux2.cfg``` is created, then installed into ```/usr/lib/python2.7/site-packages/pythran/pythran-linux2.cfg```

**Comment by serge-sans-paille** [comment]
what about 

```
pip install git+https://github.com/serge-sans-paille/pythran.git
```

**Comment by pbrunet** [comment]
I have the same issue on a fresh install from ZIP without GMP support

**Comment by pbrunet** [comment]
Reason is : we replace content to remove GMP flags but new content is smaller than older one so only the first part of the config file is correctly set and garbage is still present at the end.

**Comment by serge-sans-paille** [comment]
On Sun, Jun 19, 2016 at 11:47:18PM -0700, pbrunet wrote:
> Reason is : we replace content to remove GMP flags but new content is smaller than older one so only the first part of the config file is correctly set and garbage is still present at the end.

@pbrunet: one solution would be to replace with spaces instead of
changing size?

**Comment by pbrunet** [comment]
or just fully rewrite the conf file. I will prepare a commit with this. I just don't know how can we test it? Rerun an installation without GMP doesn't looks to be a viable solution :s

**Comment by serge-sans-paille** [comment]
Good news: @pbrunet fixed this in #582
